pkg/monitoring/metrics/virt-controller/vmstats_collector.go

```diff
@@ -480,6 +522,38 @@ func collectCpuResourceLimitsFromDomainResources(vm *k6tv1.VirtualMachine) []ope
 	return cr
 }
 
+// collectAllocatedCpuValues calculates the allocated CPU requests and limits after applying
+// instance types, preferences, defaults and hierarchy - simplified to only report the allocated values
+func collectAllocatedCpuValues(vm *k6tv1.VirtualMachine) []operatormetrics.CollectorResult {
+	var cr []operatormetrics.CollectorResult
+
+	if vm.Spec.Template == nil {
+		return cr
+	}
+
+	// Report only the allocated guest vCPUs - what the VM guest OS will actually see
+	if vm.Spec.Template.Spec.Domain.CPU != nil {
+		allocatedVCPUs := hardware.GetNumberOfVCPUs(vm.Spec.Template.Spec.Domain.CPU)
+
+		cr = append(cr, operatormetrics.CollectorResult{
+			Metric: vmResourceRequests,
+			Value:  float64(allocatedVCPUs),
+			Labels: []string{vm.Name, vm.Namespace, "cpu", "cores", "allocated"},
```

Comment:
The phrasing of "allocated" is a bit confusing IMO.
Can we change this to something like "virt-launcher resources", to ensure we make it clear the resources are not allocated to the guest itself.

Reply:
Updated to "guest_effective"